doc/userguide/file-extraction/file-extraction.rst

```diff
@@ -4,12 +4,18 @@ File Extraction
 Architecture
 ~~~~~~~~~~~~
 
-The file extraction code works on top of the HTTP and SMTP parsers. The HTTP parser takes care of dechunking and unzipping the request and/or response data if necessary. The HTTP/SMTP parsers runs on top of the stream reassembly engine.
+The file extraction code works on top of the some protocols parsers. The HTTP parser takes care of dechunking and unzipping the request and/or response data if necessary. The application layer parsers runs on top of the stream reassembly engine.
```

Comment:
**top of the some protocols parsers** I would remove the **the** or write **on top of some of the protocol parsers** :)